app/controllers/mou_controller.rb

```diff
@@ -0,0 +1,22 @@
+class MouController < ApplicationController
+  def index; end
+
+  def update
+    Organisation.find(params[:id]).signed_mou.attach(params[:signed_mou])
+    redirect_to organisations_path
+  end
+
+  def upload
+    current_organisation.signed_mou.attach(params[:signed_mou])
+    redirect_to mou_index_path
+  end
+
+  def admin
```

Comment:
Could this be named better?

Reply:
Maybe it should be a different controller? Think it'll make it clearer how separate it is, and separate out the access logic (when we come to add it in) nicely.